services/app-api/src/handlers/third_party_API_authorizer.ts

```diff
@@ -77,7 +81,8 @@ export const main: APIGatewayTokenAuthorizerHandler = async (
 
 const generatePolicy = function (
     userId: string | undefined,
-    event: APIGatewayTokenAuthorizerEvent
+    event: APIGatewayTokenAuthorizerEvent,
+    context?: Record<string, string>
```

Comment:
Could you add a comment explaining what `context` should be here. Also, if I'm understanding correctly, context has the `oathClient` info. What are the implications of having this be an optional field here? Do we need error handling or a fallback if context is missing?